src/Command/Argument/Hash/HGetFArguments.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Predis package.
+ *
+ * (c) 2009-2020 Daniele Alessandri
+ * (c) 2021-2023 Till Krüss
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Predis\Command\Argument\Hash;
+
+use UnexpectedValueException;
+
+class HGetFArguments extends CommonExpiration
+{
+    /**
+     * Removes current expiration time from given fields.
+     *
+     * @return $this
+     */
+    public function setPersist(): self
+    {
+        if (!empty(array_intersect($this->ttlModifierEnum, $this->arguments))) {
```

Comment:
What if someone first calls `setPersist` and then calls `setTtlModifier`?

Reply:
Okay, let's add the same kind of check on the other side